test/service/test_response.py

```diff
@@ -2599,9 +2605,9 @@ def test_summary_cell_counts(self):
         response.raise_for_status()
         summary = response.json()
         self.assertEqual(summary['projectCount'], 1)
```

Comment:
Can we assert assay type?

Reply:
I've added a test case for the protocol values, however we currently suffer from a lack of test data.

Not only did the recent schema changes rename `imaging_protocol`'s `target`  to `probe`, it also made `probe` [optional](https://schema.staging.data.humancellatlas.org/type/protocol/imaging/11.3.0/imaging_protocol). The bundles from the canned staging area do have an [imaging protocol](https://github.com/HumanCellAtlas/schema-test-data/blob/master/tests/metadata/imaging_protocol/50150415-898c-5dff-826e-55f9cb937b93_2021-01-01T00:00:00.000000Z.json) but do not include a `probe` (or `target`) and thus no `assay type`.

The test case I added asserts the `protocols` inner entity values, which for `assayType` is `[]`.